website/docs/advanced-concepts/granular-access-control/reference/permissions.mdx

```diff
@@ -0,0 +1,717 @@
+        </tr>
+      </tbody>
+    </table>
+  </TabItem>
+</Tabs>
+
+## Make Public permission
+
+The **Make Public** permission permits users to change the access level of resources, making them available to the public. When the **Make Public** permission is assigned to a workspace or an application, it, along with the automatically assigned permissions, are inherited by the child resources. Once granted, users can enable the **Make application public** option on the _Invite Users_ modal to make an application publicly available. This option allows external users to access the app without the need to log in to Appsmith.
+
+<Tabs queryString="gac-resource">
+  <TabItem label="Application Resources" value="make-public-application-resources">
+    <table className="permissions-table" style={{ width: '100%', display: 'table', marginLeft: '2px' }}>
+      <thead>
+        <tr>
+          <th>Resource</th>
+          <th>Make Public also grants below permissions</th>
+          <th>Permission Impact</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td><b>Workspace</b></td>
+          <td>View, Execute</td>
+          <td>Allows making all applications publicly accessible within the given workspace, and allows viewing and running queries to view the application when applications are publicly accessible.</td>
+        </tr>
+        <tr>
+          <td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Application</td>
+          <td>View, Execute</td>
+          <td>Allows making the given application publicly accessible, and allows viewing and running queries to view the application when it's publicly accessible.</td>
+        </tr>
+      </tbody>
+    </table>
+  </TabItem>
+</Tabs>
+
+## Export permission
+
+The **Export** permission permits users to export data and resources from Appsmith. When the **Export** permission is assigned to a workspace or an application, it, along with the automatically assigned permissions, are inherited by the child resources.
+
+<Tabs queryString="gac-resource">
+  <TabItem label="Application Resources" value="execute-application-resources">
+    <table className="permissions-table" style={{ width: '100%', display: 'table', marginLeft: '2px' }}>
+      <thead>
+        <tr>
+          <th>Resource</th>
+          <th>Export also grants below permissions</th>
+          <th>Permission Impact</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td><b>Workspace</b></td>
+          <td>View, Execute</td>
+          <td>Allows viewing application, exporting data and resources, and running queries required to view application and exporting data within the given workspace.</td>
+        </tr>
+        <tr>
+          <td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Application</td>
+          <td>View, Execute</td>
+          <td>Allows viewing application, exporting data and resources, and running queries required to view application and exporting data within the given application.</td>
```

Comment:
Export permission is mainly to export an application and that's it, we should not explain about lateral permissions.

Reply:
We had many user queries - users struggled with understanding why when they assign permission, others are assigned too, and their implications. We added a description accordingly. I will try to rewrite - to showcase primary permission - and then describe lateral permission.

Reply:
Yeah, May be add a section on lateral permissions in permissions page to cover this more generally in that case.

Reply:
I've rewritten the descriptions of permission impact. IMO, that should help with the understanding. Let me know if that doesn't work.